src/scripts/export-page.js

```diff
@@ -1,5 +1,6 @@
 import { saveAs } from "file-saver";
 import { HeadingLevel, Paragraph, Document, TextRun, Packer } from "docx";
+import { isMobile } from "react-device-detect";

```

Comment:
This strikes me as odd:

Why react-device-detect in a non-react subproject?

Looks like react-device-detect is _just_ a react layer on top of ua-parser-js.

How about 

import { getDevice } from 'ua-parser-js'; 

coupled with

['mobile', 'tablet'].includes(getDevice().type)